spec/rspec/rails/fixture_support_spec.rb

```diff
@@ -12,6 +12,33 @@ module RSpec::Rails
       end
     end
 
+    context "with use_transactional_tests set to true" do
+      it "works with #uses_transaction helper" do
+        group = RSpec::Core::ExampleGroup.describe do
+          include FixtureSupport
+          self.use_transactional_tests = true
+
+          uses_transaction "doesn't run in transaction"
+
+          it "doesn't run in transaction" do
+            expect(run_in_transaction?).to eq(false)
+          end
+
+          it "runs in transaction" do
+            expect(run_in_transaction?).to eq(true)
+          end
```

Comment:
What do you think of changing those expectations to:
```ruby
expect(ActiveRecord::Base.connection.transaction_open?).to eq(...)
```
Worked for me.
It feels a bit more reassuring.

Reply:
Good point 👍 have updated

Reply:
it looks like the change caused some builds to fail but I can't see the log. can you gather some error messages here?